setup.py

```diff
@@ -2,13 +2,20 @@
 # -*- coding: utf-8 -*-
 
 from setuptools import setup, find_packages
+from distutils.util import convert_path
+
+main_ns = {}
+ver_path = convert_path('scvi/__init__.py')
+with open(ver_path) as ver_file:
+    exec(ver_file.read(), main_ns)
```

Comment:
I don't get it...why are we changing setup.py at all in this pull request?

Reply:
It will be easier to update the version number (when I updated to `pip`). We need to change `version` in two places (`setup.py` and `scvi.__init__.py`) before. Now we only need to change it in `scvi.__init__.py`. 😊 But I can change it back haha. Sorry about this 😔

Reply:
Yes, better to change it back. Just also use `bumpversion` to change the version numbers. It will change them everywhere.

Reply:
Thanks!